.coveragerc

```diff
@@ -1158,6 +1158,9 @@ omit =
     homeassistant/components/upcloud/switch.py
     homeassistant/components/upnp/*
     homeassistant/components/upc_connect/*
+    homeassistant/components/uptime_kuma/__init__.py
+    homeassistant/components/uptime_kuma/const.py
+    homeassistant/components/uptime_kuma/binary_sensor.py
```

Comment:
Though it is not compulsory, I suggest that you try to get 100% coverage as you wait for the PR to get fully reviewed.

Reply:
Unfortunately i'm just not up to speed on testing, so I won't be able to do this.